test_shaders.py

```diff
@@ -459,19 +557,35 @@ def main():
     parser.add_argument('--opt',
             action = 'store_true',
             help = 'Run SPIRV-Tools optimization passes as well.')
+    parser.add_argument('--reflect',
+            action = 'store_true',
+            help = 'Test reflection backend.')
+    parser.add_argument('--parallel',
+            action = 'store_true',
+            help = 'Execute tests in parallel.  Useful for doing regression quickly, but bad for debugging and stat output.')
+    
+    global args;
```

Comment:
semicolon?

Reply:
curse my 20 years of C++ & Java reflexes.